pages/src/navigation/header/header.less

```diff
@@ -7,6 +7,8 @@
   background-color: @landing-bg-color;
 
   max-height: 56px;
+
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, .5);
```

Comment:
Unfortunately, we need to rethink this approach because right now shadow doesn't look good on the landing page with the marquee. We should not display shadow until we scroll the page.

We need to think about possible solutions. My current idea is to create a shadow extra element after the main section scroll so it can consume `at-start` marker. And display it with position fixed under the header to produce a shadow effect.